staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -310,52 +308,6 @@ var (
 			},
 		},
 	)
-	// We add an explicit rule for endpoint-controller with high precedence
```

Comment:
This removal I believe is fine - this will encourage folks using endpoints-based leader election to actually migrate.